others/paranthesis_matching.cpp

```diff
@@ -20,13 +20,13 @@
 char stack[MAX];
 
 //! pointer to track stack index
-int top = -1;
+int top_var = -1;
```

Comment:
maybe rename this as 
```suggestion
int stack_idx = -1;
```